test/config/monitoring/monitoring.yaml

```
@@ -43,7 +43,7 @@ spec:
     spec:
       containers:
       - name: zipkin
-        image: docker.io/openzipkin/zipkin:2.23.2
+        image: ghcr.io/openzipkin/zipkin:latest
```

```suggestion
        image: ghcr.io/openzipkin/zipkin:2
```

Maybe safer would be to use: `ghcr.io/openzipkin/zipkin:2` instead? If they release zipkin 3 that breaks something, this should save us from failure. Or even `2.23` tag to feel super safe.